google/iam/admin/v1/iam_gapic.yaml

```diff
@@ -0,0 +1,248 @@
+type: com.google.api.codegen.ConfigProto
+language_settings:
+  java:
+    package_name: com.google.cloud.iam.admin.spi.v1
+  python:
+    package_name: google.cloud.iam.admin.v1
```

Comment:
Is that right?  Want @bjwatson or @geigerj  to review -- possibly need 'gapic', like `google.cloud.gapic.iam.admin.v1`?